### ISSUE TITLE
docs(cdk): add API docs for a11y region attributes

### ISSUE THREAD

**Issue by Splaktar** [angular/components#15799]
#### What is the expected behavior?
The 3 attributes discussed in https://material.angular.io/cdk/a11y/overview#regions (`cdkFocusRegionStart`, `cdkFocusRegionEnd` and `cdkFocusInitial`) should have API docs.

#### What is the current behavior?
These APIs are not found on https://material.angular.io/cdk/a11y/api.

#### What are the steps to reproduce?
See above.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular Material: 7.3.7

#### Is there anything else we should know?
No.

**Comment by Splaktar** [comment]
Ah yeah, good point. I don't see a good way for dgeni to discover and document these either 😞 
https://github.com/angular/components/blob/48c70da05bfe8d542c4c76433191ea91ac98239f/src/cdk/a11y/focus-trap/focus-trap.ts#L167-L169
